src/components/LandingPage/Navigation.js

```diff
@@ -0,0 +1,78 @@
+import React, { Component } from 'react'
+import { BrowserRouter as Router,Link as Links} from "react-router-dom";
+import { animateScroll as scroll , Link} from 'react-scroll'
+import $ from 'jquery';
+import './../../Assets/Styles/Navigation.scss'
+
+ $(document).ready(function() {
+            $(".menu-icon").on("click", function() {
+                  $("nav ul").toggleClass("showing");
+            });
+      });
+
+      // Scrolling Effect
+
+ $(window).on("scroll", function() {
+            if($(window).scrollTop()) {
+                  $('nav').addClass('black');
+            }
+
+            else {
+                  $('nav').removeClass('black');
+            }
+      })
```

Comment:
Don't mix jquery with react. Refactor to have the same functionality with react.